test-long CI failure with "Killed due to illegal instruction"

here are some preliminary investigations.

* the illegal instruction is inside `/sage/local/lib/libffpack.so.1` or `/sage/local/lib/libfflas.so.1`.

* some sample stack traces

```
2025-10-06T23:57:21.4432036Z ##[error]/sage/local/var/lib/sage/venv-python3.12/lib/python3.12/site-packages/cysignals/signals.cpython-312-x86_64-linux-gnu.so(+0x96a4)[0x7fc9b96ad6a4]
/sage/local/var/lib/sage/venv-python3.12/lib/python3.12/site-packages/cysignals/signals.cpython-312-x86_64-linux-gnu.so(+0x9766)[0x7fc9b96ad766]
/sage/local/var/lib/sage/venv-python3.12/lib/python3.12/site-packages/cysignals/signals.cpython-312-x86_64-linux-gnu.so(+0xc484)[0x7fc9b96b0484]
/lib/x86_64-linux-gnu/libc.so.6(+0x45330)[0x7fc9b9c48330]
/sage/local/lib/libfflas.so.1(+0x2b8f7)[0x7fc9653b98f7]
/sage/local/lib/libfflas.so.1(_ZN5FFLAS5fgemvIN6Givaro7ModularIddvEEEENT_11Element_ptrERKS4_NS_15FFLAS_TRANSPOSEEmmNS4_7ElementENS4_16ConstElement_ptrEmSA_mS9_S5_m+0x1ba)[0x7fc9653cc0aa]
/sage/local/lib/libffpack.so.1(_ZN6FFPACK18PLUQ_basecaseCroutIN6Givaro7ModularIddvEEEEmRKT_N5FFLAS10FFLAS_DIAGEmmNS4_11Element_ptrEmPmSA_+0x2b8)[0x7fc9653084d8]
```

* sometimes it fail with an exception say
```
sage: charpoly = g.characteristic_polynomial() ## line 79 ##

2025-10-06T23:59:24.1126850Z Traceback (most recent call last):
2025-10-06T23:59:24.1127393Z   File "/sage/src/sage/doctest/forker.py", line 2642, in __call__
2025-10-06T23:59:24.1128028Z     doctests, extras = self._run(runner, options, results)
2025-10-06T23:59:24.1128525Z                        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2025-10-06T23:59:24.1129049Z   File "/sage/src/sage/doctest/forker.py", line 2690, in _run
2025-10-06T23:59:24.1129764Z     result = runner.run(test)
2025-10-06T23:59:24.1130135Z              ^^^^^^^^^^^^^^^^
2025-10-06T23:59:24.1130555Z   File "/sage/src/sage/doctest/forker.py", line 925, in run
2025-10-06T23:59:24.1131101Z     return self._run(test, compileflags, out)
2025-10-06T23:59:24.1131527Z            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2025-10-06T23:59:24.1132012Z   File "/sage/src/sage/doctest/forker.py", line 733, in _run
2025-10-06T23:59:24.1132611Z     self.compile_and_execute(example, compiler, test.globs)
2025-10-06T23:59:24.1133287Z   File "/sage/src/sage/doctest/forker.py", line 1157, in compile_and_execute
2025-10-06T23:59:24.1133882Z     exec(compiled, globs)
2025-10-06T23:59:24.1134435Z   File "<doctest sage.graphs.generators.basic.BullGraph[8]>", line 1, in <module>
2025-10-06T23:59:24.1135313Z   File "/sage/src/sage/graphs/generic_graph.py", line 23860, in characteristic_polynomial
2025-10-06T23:59:24.1136120Z     return self.adjacency_matrix(vertices=list(self)).charpoly(var=var)
2025-10-06T23:59:24.1136706Z            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2025-10-06T23:59:24.1137565Z   File "sage/matrix/matrix_integer_dense.pyx", line 1416, in sage.matrix.matrix_integer_dense.Matrix_integer_dense.charpoly
2025-10-06T23:59:24.1138479Z cysignals.signals.SignalError: Illegal instruction
```

but this looks like just because of the `sig_on()` being there.

* coincidentally https://github.com/sagemath/sage/pull/40975 touches this part of code, but I hope it's unrelated because e.g. `rank` also fail the same way
```
2025-10-07T00:02:48.7092797Z   File "/sage/src/sage/modules/free_module.py", line 7722, in __init__
2025-10-07T00:02:48.7093527Z     FreeModule_submodule_with_basis_pid.__init__(self, ambient, basis=gens,
2025-10-07T00:02:48.7094261Z   File "/sage/src/sage/modules/free_module.py", line 6689, in __init__
2025-10-07T00:02:48.7094860Z     basis = self._echelonized_basis(ambient, basis)
2025-10-07T00:02:48.7095309Z             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2025-10-07T00:02:48.7095909Z   File "/sage/src/sage/modules/free_module.py", line 6882, in _echelonized_basis
2025-10-07T00:02:48.7096502Z     r = E.rank()
2025-10-07T00:02:48.7096789Z         ^^^^^^^^
2025-10-07T00:02:48.7097517Z   File "sage/matrix/matrix_integer_sparse.pyx", line 715, in sage.matrix.matrix_integer_sparse.Matrix_integer_sparse.rank
2025-10-07T00:02:48.7098770Z   File "sage/matrix/matrix_integer_sparse.pyx", line 756, in sage.matrix.matrix_integer_sparse.Matrix_integer_sparse._rank_linbox
```

* the test-long for the 10.8.beta6 push itself succeed https://github.com/sagemath/sage/actions/runs/18294000943/job/52087930463 (`logs_47031098017.zip`)

```
2025-10-06T21:16:59.7415515Z #25 16.06 brial:                          installed SPKG version 1.2.14 will be replaced by version 1.2.15

2025-10-06T21:17:00.2558833Z #25 16.53 fflas_ffpack:                   SPKG version 2.5.0+sage-2024-05-18b is already installed

2025-10-06T21:17:02.1238017Z #25 18.38 sage_conf:                      standard, installed SPKG version 10.8.beta5 will be replaced by version 10.8.beta6
```

* the failing tests are based on the following docker image

```
2025-10-06T23:41:07.6403205Z #4 [with-system-packages 1/4] FROM ghcr.io/sagemath/sage/sage-ubuntu-noble-standard-with-targets:dev@sha256:da748ffab8cbca0f7b511e7389feea29529eff975c9b3765e878c80e24811338
```

* which get pushed by https://github.com/sagemath/sage/actions/runs/18293999539 (Ubuntu-noble) (`logs_47031094310.zip`)

```
2025-10-06T21:12:42.0562247Z make --no-print-directory fflas_ffpack-SAGE_LOCAL-no-deps
2025-10-06T21:12:42.2257537Z [fflas_ffpack-2.5.0+sage-2024-05-18b] installing. Log file: /sage/logs/pkgs/fflas_ffpack-2.5.0+sage-2024-05-18b.log
2025-10-06T21:13:49.1391721Z   [brial-1.2.15] successfully installed (real 3m12.371s user 1m56.068s sys 0m7.398s).
2025-10-06T21:14:22.4875545Z   [fplll-5.5.0] successfully installed (real 12m21.748s user 15m11.499s sys 0m16.509s).
2025-10-06T21:16:29.6029940Z   [singular-4.4.1] successfully installed (real 14m22.116s user 8m18.844s sys 0m39.122s).
2025-10-06T21:23:19.8649032Z   [fflas_ffpack-2.5.0+sage-2024-05-18b] successfully installed (real 10m37.630s user 22m8.814s sys 0m36.966s).

2025-10-06T23:32:38.9255155Z ##[notice]Pushed images ghcr.io/sagemath/sage/sage-ubuntu-noble-standard-with-system-packages:4e2319bc162 ghcr.io/sagemath/sage/sage-ubuntu-noble-standard-with-system-packages:dev ghcr.io/sagemath/sage/sage-ubuntu-noble-standard-with-system-packages:10.8.beta6 ghcr.io/sagemath/sage/sage-ubuntu-noble-standard-configured:4e2319bc162 ghcr.io/sagemath/sage/sage-ubuntu-noble-standard-configured:dev ghcr.io/sagemath/sage/sage-ubuntu-noble-standard-configured:10.8.beta6 ghcr.io/sagemath/sage/sage-ubuntu-noble-standard-with-targets-pre:4e2319bc162 ghcr.io/sagemath/sage/sage-ubuntu-noble-standard-with-targets-pre:dev ghcr.io/sagemath/sage/sage-ubuntu-noble-standard-with-targets-pre:10.8.beta6 ghcr.io/sagemath/sage/sage-ubuntu-noble-standard-with-targets:4e2319bc162 ghcr.io/sagemath/sage/sage-ubuntu-noble-standard-with-targets:dev ghcr.io/sagemath/sage/sage-ubuntu-noble-standard-with-targets:10.8.beta6 ghcr.io/sagemath/sage/sage-ubuntu-noble-standard-with-targets-optional:4e2319bc162 ghcr.io/sagemath/sage/sage-ubuntu-noble-standard-with-targets-optional:dev ghcr.io/sagemath/sage/sage-ubuntu-noble-standard-with-targets-optional:10.8.beta6)
2025-10-06T23:32:38.9265478Z 
2025-10-06T23:32:38.9265731Z To pull one of the images and enter the container, type:
2025-10-06T23:32:38.9265947Z 
2025-10-06T23:32:38.9266235Z   $ docker run -it ghcr.io/sagemath/sage/sage-ubuntu-noble-standard-with-system-packages:4e2319bc162 bash
2025-10-06T23:32:38.9267071Z   $ docker run -it ghcr.io/sagemath/sage/sage-ubuntu-noble-standard-with-system-packages:dev bash
2025-10-06T23:32:38.9267661Z   $ docker run -it ghcr.io/sagemath/sage/sage-ubuntu-noble-standard-with-system-packages:10.8.beta6 bash
2025-10-06T23:32:38.9268236Z   $ docker run -it ghcr.io/sagemath/sage/sage-ubuntu-noble-standard-configured:4e2319bc162 bash
2025-10-06T23:32:38.9268745Z   $ docker run -it ghcr.io/sagemath/sage/sage-ubuntu-noble-standard-configured:dev bash
2025-10-06T23:32:38.9269261Z   $ docker run -it ghcr.io/sagemath/sage/sage-ubuntu-noble-standard-configured:10.8.beta6 bash
2025-10-06T23:32:38.9269815Z   $ docker run -it ghcr.io/sagemath/sage/sage-ubuntu-noble-standard-with-targets-pre:4e2319bc162 bash
2025-10-06T23:32:38.9270380Z   $ docker run -it ghcr.io/sagemath/sage/sage-ubuntu-noble-standard-with-targets-pre:dev bash
2025-10-06T23:32:38.9270930Z   $ docker run -it ghcr.io/sagemath/sage/sage-ubuntu-noble-standard-with-targets-pre:10.8.beta6 bash
2025-10-06T23:32:38.9271496Z   $ docker run -it ghcr.io/sagemath/sage/sage-ubuntu-noble-standard-with-targets:4e2319bc162 bash
2025-10-06T23:32:38.9272016Z   $ docker run -it ghcr.io/sagemath/sage/sage-ubuntu-noble-standard-with-targets:dev bash
2025-10-06T23:32:38.9272525Z   $ docker run -it ghcr.io/sagemath/sage/sage-ubuntu-noble-standard-with-targets:10.8.beta6 bash
2025-10-06T23:32:38.9273103Z   $ docker run -it ghcr.io/sagemath/sage/sage-ubuntu-noble-standard-with-targets-optional:4e2319bc162 bash
2025-10-06T23:32:38.9273701Z   $ docker run -it ghcr.io/sagemath/sage/sage-ubuntu-noble-standard-with-targets-optional:dev bash
2025-10-06T23:32:38.9274288Z   $ docker run -it ghcr.io/sagemath/sage/sage-ubuntu-noble-standard-with-targets-optional:10.8.beta6 bash
2025-10-06T23:32:38.9274798Z 
2025-10-06T23:32:38.9274950Z To use one of the images as the base for an incremental build, type:
2025-10-06T23:32:38.9275181Z 
2025-10-06T23:32:38.9275581Z   $ FROM_DOCKER_TARGET=with-system-packages FROM_DOCKER_TAG=4e2319bc162 tox -e docker-ubuntu-noble-standard-incremental
2025-10-06T23:32:38.9276246Z   $ FROM_DOCKER_TARGET=with-system-packages FROM_DOCKER_TAG=dev tox -e docker-ubuntu-noble-standard-incremental
2025-10-06T23:32:38.9277061Z   $ FROM_DOCKER_TARGET=with-system-packages FROM_DOCKER_TAG=10.8.beta6 tox -e docker-ubuntu-noble-standard-incremental
2025-10-06T23:32:38.9277689Z   $ FROM_DOCKER_TARGET=configured FROM_DOCKER_TAG=4e2319bc162 tox -e docker-ubuntu-noble-standard-incremental
2025-10-06T23:32:38.9278287Z   $ FROM_DOCKER_TARGET=configured FROM_DOCKER_TAG=dev tox -e docker-ubuntu-noble-standard-incremental
2025-10-06T23:32:38.9278870Z   $ FROM_DOCKER_TARGET=configured FROM_DOCKER_TAG=10.8.beta6 tox -e docker-ubuntu-noble-standard-incremental
2025-10-06T23:32:38.9279492Z   $ FROM_DOCKER_TARGET=with-targets-pre FROM_DOCKER_TAG=4e2319bc162 tox -e docker-ubuntu-noble-standard-incremental
2025-10-06T23:32:38.9280097Z   $ FROM_DOCKER_TARGET=with-targets-pre FROM_DOCKER_TAG=dev tox -e docker-ubuntu-noble-standard-incremental
2025-10-06T23:32:38.9280707Z   $ FROM_DOCKER_TARGET=with-targets-pre FROM_DOCKER_TAG=10.8.beta6 tox -e docker-ubuntu-noble-standard-incremental
2025-10-06T23:32:38.9281240Z   $ FROM_DOCKER_TAG=4e2319bc162 tox -e docker-ubuntu-noble-standard-incremental
2025-10-06T23:32:38.9281666Z   $ FROM_DOCKER_TAG=dev tox -e docker-ubuntu-noble-standard-incremental
2025-10-06T23:32:38.9282060Z   $ FROM_DOCKER_TAG=10.8.beta6 tox -e docker-ubuntu-noble-standard-incremental
2025-10-06T23:32:38.9282609Z   $ FROM_DOCKER_TARGET=with-targets-optional FROM_DOCKER_TAG=4e2319bc162 tox -e docker-ubuntu-noble-standard-incremental
2025-10-06T23:32:38.9283260Z   $ FROM_DOCKER_TARGET=with-targets-optional FROM_DOCKER_TAG=dev tox -e docker-ubuntu-noble-standard-incremental
2025-10-06T23:32:38.9283901Z   $ FROM_DOCKER_TARGET=with-targets-optional FROM_DOCKER_TAG=10.8.beta6 tox -e docker-ubuntu-noble-standard-incremental
2025-10-06T23:32:38.9335721Z Post job cleanup.
```

* from the log artifact: 

[fflas_ffpack-2.5.0+sage-2024-05-18b.log](https://github.com/user-attachments/files/22748830/fflas_ffpack-2.5.0%2Bsage-2024-05-18b.log)

[installed__fflas_ffpack-2.5.0+sage-2024-05-18b.log](https://github.com/user-attachments/files/22748866/installed__fflas_ffpack-2.5.0%2Bsage-2024-05-18b.log)

```
[spkg-install] /usr/bin/bash ../../../libtool  --tag=CXX   --mode=compile g++ -std=gnu++11 -DHAVE_CONFIG_H -I. -I../../..  -I../../..  -O2 -march=native -Wall -DNDEBUG -UDEBUG  -I/usr/include/x86_64-linux-gnu/openblas-pthread/   -g -O2 -c -o fflas_L1_inst.lo fflas_L1_inst.C
[spkg-install] libtool: compile:  g++ -std=gnu++11 -DHAVE_CONFIG_H -I. -I../../.. -I../../.. -O2 -march=native -Wall -DNDEBUG -UDEBUG -I/usr/include/x86_64-linux-gnu/openblas-pthread/ -g -O2 -c fflas_L1_inst.C  -fPIC -DPIC -o .libs/fflas_L1_inst.o
[spkg-install] /usr/bin/bash ../../../libtool  --tag=CXX   --mode=compile g++ -std=gnu++11 -DHAVE_CONFIG_H -I. -I../../..  -I../../..  -O2 -march=native -Wall -DNDEBUG -UDEBUG  -I/usr/include/x86_64-linux-gnu/openblas-pthread/   -g -O2 -c -o fflas_L2_inst.lo fflas_L2_inst.C
[spkg-install] libtool: compile:  g++ -std=gnu++11 -DHAVE_CONFIG_H -I. -I../../.. -I../../.. -O2 -march=native -Wall -DNDEBUG -UDEBUG -I/usr/include/x86_64-linux-gnu/openblas-pthread/ -g -O2 -c fflas_L2_inst.C  -fPIC -DPIC -o .libs/fflas_L2_inst.o
[spkg-install] /usr/bin/bash ../../../libtool  --tag=CXX   --mode=compile g++ -std=gnu++11 -DHAVE_CONFIG_H -I. -I../../..  -I../../..  -O2 -march=native -Wall -DNDEBUG -UDEBUG  -I/usr/include/x86_64-linux-gnu/openblas-pthread/   -g -O2 -c -o fflas_L3_inst.lo fflas_L3_inst.C
[spkg-install] libtool: compile:  g++ -std=gnu++11 -DHAVE_CONFIG_H -I. -I../../.. -I../../.. -O2 -march=native -Wall -DNDEBUG -UDEBUG -I/usr/include/x86_64-linux-gnu/openblas-pthread/ -g -O2 -c fflas_L3_inst.C  -fPIC -DPIC -o .libs/fflas_L3_inst.o
[spkg-install] /usr/bin/bash ../../../libtool  --tag=CXX   --mode=compile g++ -std=gnu++11 -DHAVE_CONFIG_H -I. -I../../..  -I../../..  -O2 -march=native -Wall -DNDEBUG -UDEBUG  -I/usr/include/x86_64-linux-gnu/openblas-pthread/   -g -O2 -c -o ffpack_inst.lo ffpack_inst.C
[spkg-install] libtool: compile:  g++ -std=gnu++11 -DHAVE_CONFIG_H -I. -I../../.. -I../../.. -O2 -march=native -Wall -DNDEBUG -UDEBUG -I/usr/include/x86_64-linux-gnu/openblas-pthread/ -g -O2 -c ffpack_inst.C  -fPIC -DPIC -o .libs/ffpack_inst.o
[spkg-install] /usr/bin/bash ../../../libtool  --tag=CXX   --mode=compile g++ -std=gnu++11 -DHAVE_CONFIG_H -I. -I../../..  -I../../..  -O2 -march=native -Wall -DNDEBUG -UDEBUG  -I/usr/include/x86_64-linux-gnu/openblas-pthread/   -g -O2 -c -o fflas_lvl1.lo fflas_lvl1.C
[spkg-install] libtool: compile:  g++ -std=gnu++11 -DHAVE_CONFIG_H -I. -I../../.. -I../../.. -O2 -march=native -Wall -DNDEBUG -UDEBUG -I/usr/include/x86_64-linux-gnu/openblas-pthread/ -g -O2 -c fflas_lvl1.C  -fPIC -DPIC -o .libs/fflas_lvl1.o
[spkg-install] /usr/bin/bash ../../../libtool  --tag=CXX   --mode=compile g++ -std=gnu++11 -DHAVE_CONFIG_H -I. -I../../..  -I../../..  -O2 -march=native -Wall -DNDEBUG -UDEBUG  -I/usr/include/x86_64-linux-gnu/openblas-pthread/   -g -O2 -c -o fflas_lvl2.lo fflas_lvl2.C
[spkg-install] libtool: compile:  g++ -std=gnu++11 -DHAVE_CONFIG_H -I. -I../../.. -I../../.. -O2 -march=native -Wall -DNDEBUG -UDEBUG -I/usr/include/x86_64-linux-gnu/openblas-pthread/ -g -O2 -c fflas_lvl2.C  -fPIC -DPIC -o .libs/fflas_lvl2.o
[spkg-install] /usr/bin/bash ../../../libtool  --tag=CXX   --mode=compile g++ -std=gnu++11 -DHAVE_CONFIG_H -I. -I../../..  -I../../..  -O2 -march=native -Wall -DNDEBUG -UDEBUG  -I/usr/include/x86_64-linux-gnu/openblas-pthread/   -g -O2 -c -o fflas_lvl3.lo fflas_lvl3.C
[spkg-install] libtool: compile:  g++ -std=gnu++11 -DHAVE_CONFIG_H -I. -I../../.. -I../../.. -O2 -march=native -Wall -DNDEBUG -UDEBUG -I/usr/include/x86_64-linux-gnu/openblas-pthread/ -g -O2 -c fflas_lvl3.C  -fPIC -DPIC -o .libs/fflas_lvl3.o
[spkg-install] /usr/bin/bash ../../../libtool  --tag=CXX   --mode=compile g++ -std=gnu++11 -DHAVE_CONFIG_H -I. -I../../..  -I../../..  -O2 -march=native -Wall -DNDEBUG -UDEBUG  -I/usr/include/x86_64-linux-gnu/openblas-pthread/   -g -O2 -c -o fflas_sparse.lo fflas_sparse.C
[spkg-install] libtool: compile:  g++ -std=gnu++11 -DHAVE_CONFIG_H -I. -I../../.. -I../../.. -O2 -march=native -Wall -DNDEBUG -UDEBUG -I/usr/include/x86_64-linux-gnu/openblas-pthread/ -g -O2 -c fflas_sparse.C  -fPIC -DPIC -o .libs/fflas_sparse.o
[spkg-install] /usr/bin/bash ../../../libtool  --tag=CXX   --mode=compile g++ -std=gnu++11 -DHAVE_CONFIG_H -I. -I../../..  -I../../..  -O2 -march=native -Wall -DNDEBUG -UDEBUG  -I/usr/include/x86_64-linux-gnu/openblas-pthread/   -g -O2 -c -o ffpack.lo ffpack.C
[spkg-install] libtool: compile:  g++ -std=gnu++11 -DHAVE_CONFIG_H -I. -I../../.. -I../../.. -O2 -march=native -Wall -DNDEBUG -UDEBUG -I/usr/include/x86_64-linux-gnu/openblas-pthread/ -g -O2 -c ffpack.C  -fPIC -DPIC -o .libs/ffpack.o
[spkg-install] /usr/bin/bash ../../../libtool  --tag=CXX   --mode=link g++ -std=gnu++11 -O2 -march=native -Wall -DNDEBUG -UDEBUG  -I/usr/include/x86_64-linux-gnu/openblas-pthread/   -g -O2 -version-info 1:0:0 -no-undefined -Wl,-rpath-link,/sage/local/lib -L/sage/local/lib -Wl,-rpath,/sage/local/lib -Wl,-rpath-link,/sage/local/lib -L/sage/local/lib -Wl,-rpath,/sage/local/lib  -o libfflas.la -rpath /sage/local/lib fflas_L1_inst.lo fflas_L2_inst.lo fflas_L3_inst.lo -lgivaro -lgmpxx -lgmp  -L/usr/lib/x86_64-linux-gnu/openblas-pthread/ -lopenblas   -L/usr/lib/x86_64-linux-gnu/openblas-pthread/ -lopenblas 
[spkg-install] libtool: link: g++ -std=gnu++11  -fPIC -DPIC -shared -nostdlib /usr/lib/gcc/x86_64-linux-gnu/13/../../../x86_64-linux-gnu/crti.o /usr/lib/gcc/x86_64-linux-gnu/13/crtbeginS.o  .libs/fflas_L1_inst.o .libs/fflas_L2_inst.o .libs/fflas_L3_inst.o   -L/sage/local/lib -lgivaro -lgmpxx -lgmp -L/usr/lib/x86_64-linux-gnu/openblas-pthread/ -lopenblas -L/sage/local/lib/../lib -L/usr/lib/gcc/x86_64-linux-gnu/13 -L/usr/lib/gcc/x86_64-linux-gnu/13/../../../x86_64-linux-gnu -L/usr/lib/gcc/x86_64-linux-gnu/13/../../../../lib -L/lib/x86_64-linux-gnu -L/lib/../lib -L/usr/lib/x86_64-linux-gnu -L/usr/lib/../lib -L/usr/lib/gcc/x86_64-linux-gnu/13/../../.. -lstdc++ -lm -lc -lgcc_s /usr/lib/gcc/x86_64-linux-gnu/13/crtendS.o /usr/lib/gcc/x86_64-linux-gnu/13/../../../x86_64-linux-gnu/crtn.o  -O2 -march=native -g -O2 -Wl,-rpath-link -Wl,/sage/local/lib -Wl,-rpath -Wl,/sage/local/lib -Wl,-rpath-link -Wl,/sage/local/lib -Wl,-rpath -Wl,/sage/local/lib   -Wl,-soname -Wl,libfflas.so.1 -o .libs/libfflas.so.1.0.0
```

* the test in https://github.com/sagemath/sage/actions/runs/18293999539/job/52087926437 itself run doctest, which succeed
* looking at the docker build job

```
Trying to install fflas-ffpack
debconf: delaying package configuration, since apt-utils is not installed
Selecting previously unselected package fflas-ffpack-common.
(Reading database ... 
(Reading database ... 5%
(Reading database ... 10%
(Reading database ... 15%
(Reading database ... 20%
(Reading database ... 25%
(Reading database ... 30%
(Reading database ... 35%
(Reading database ... 40%
(Reading database ... 45%
(Reading database ... 50%
(Reading database ... 55%
(Reading database ... 60%
(Reading database ... 65%
(Reading database ... 70%
(Reading database ... 75%
(Reading database ... 80%
(Reading database ... 85%
(Reading database ... 90%
(Reading database ... 95%
(Reading database ... 100%
(Reading database ... 36639 files and directories currently installed.)
Preparing to unpack .../fflas-ffpack-common_2.5.0-3_all.deb ...
Unpacking fflas-ffpack-common (2.5.0-3) ...
Selecting previously unselected package libgivaro9:amd64.
Preparing to unpack .../libgivaro9_4.2.0-4build1_amd64.deb ...
Unpacking libgivaro9:amd64 (4.2.0-4build1) ...
Selecting previously unselected package libgivaro-dev.
Preparing to unpack .../libgivaro-dev_4.2.0-4build1_amd64.deb ...
Unpacking libgivaro-dev (4.2.0-4build1) ...
Selecting previously unselected package fflas-ffpack.
Preparing to unpack .../fflas-ffpack_2.5.0-3_amd64.deb ...
Unpacking fflas-ffpack (2.5.0-3) ...
Setting up libgivaro9:amd64 (4.2.0-4build1) ...
Setting up fflas-ffpack-common (2.5.0-3) ...
Setting up libgivaro-dev (4.2.0-4build1) ...
Setting up fflas-ffpack (2.5.0-3) ...
Processing triggers for libc-bin (2.39-0ubuntu8.6) ...
Trying to install libflint-dev
debconf: delaying package configuration, since apt-utils is not installed
Selecting previously unselected package libflint18t64:amd64.
(Reading database ... 
(Reading database ... 5%
(Reading database ... 10%
(Reading database ... 15%
(Reading database ... 20%
(Reading database ... 25%
(Reading database ... 30%
(Reading database ... 35%
(Reading database ... 40%
(Reading database ... 45%
(Reading database ... 50%
(Reading database ... 55%
(Reading database ... 60%
(Reading database ... 65%
(Reading database ... 70%
(Reading database ... 75%
(Reading database ... 80%
(Reading database ... 85%
(Reading database ... 90%
(Reading database ... 95%
(Reading database ... 100%
(Reading database ... 37161 files and directories currently installed.)
```

these are identical between the docker build for 10.8.beta5 and 10.8.beta6.

* Possibly related https://github.com/sagemath/sage/issues/40699, https://github.com/sagemath/sage/issues/32434

--- a/build/pkgs/fflas_ffpack/spkg-install.in
+++ b/build/pkgs/fflas_ffpack/spkg-install.in
@@ -20,7 +20,7 @@ FFLAS_FFPACK_CONFIGURE="$SAGE_CONFIGURE_FFLAS_FFPACK $FFLAS_FFPACK_CONFIGURE"
 
 # If SAGE_FAT_BINARY is set, disable all processor-specific optimizations
 if [ "$SAGE_FAT_BINARY" = yes ]; then
-    FFLAS_FFPACK_CONFIGURE="--disable-sse --disable-sse2 --disable-sse3 --disable-ssse3 --disable-sse41 --disable-sse42 --disable-fma --disable-fma4 --disable-avx --disable-avx2 --disable-avx512f --disable-avx512dq --disable-avx512vl $FFLAS_FFPACK_CONFIGURE"
+    FFLAS_FFPACK_CONFIGURE="--without-archnative $FFLAS_FFPACK_CONFIGURE"
 fi
 
 # Need to use 'bash' for configure, see

--- a/build/pkgs/givaro/spkg-install.in
+++ b/build/pkgs/givaro/spkg-install.in
@@ -6,7 +6,7 @@ cd src
 
 # When SAGE_FAT_BINARY is set, disable processor-specific optimizations
 if [ "$SAGE_FAT_BINARY" = yes ]; then
-    GIVARO_CONFIGURE="--disable-sse --disable-sse2 --disable-sse3 --disable-ssse3 --disable-sse41 --disable-sse42 --disable-fma --disable-fma4 --disable-avx --disable-avx2  $GIVARO_CONFIGURE"
+    GIVARO_CONFIGURE="--without-archnative $GIVARO_CONFIGURE"
 fi
 
 cp "$SAGE_ROOT"/config/config.* build-aux/

--- a/build/pkgs/linbox/spkg-install.in
+++ b/build/pkgs/linbox/spkg-install.in
@@ -15,7 +15,7 @@ export CPPFLAGS="$CPPFLAGS -DDISABLE_COMMENTATOR"
 
 # If SAGE_FAT_BINARY is set, disable dependency that be discovered on the building system.
 if [ "$SAGE_FAT_BINARY" = yes ]; then
-    LINBOX_CONFIGURE="--disable-sse3 --disable-ssse3 --disable-sse41 --disable-sse42 --disable-fma --disable-fma4 --disable-avx --disable-avx2 --without-ocl $LINBOX_CONFIGURE"
+    LINBOX_CONFIGURE="--without-archnative --without-ocl $LINBOX_CONFIGURE"
 fi
 
 # Disable fplll as version 5.x is not supported by linbox <= 1.5.0.

